snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -18,8 +18,8 @@ import XCTest
 var deviceLanguage = ""
 var locale = ""
 
-func setupSnapshot(_ app: XCUIApplication) {
-    Snapshot.setupSnapshot(app)
+func setupSnapshot(_ app: XCUIApplication, animationsEnabled: Bool = true) {
```

Comment:
Can we rename this to something like `waitForAnimations`? The current name seems like it will enable/disable all animations in the app 🙃

Reply:
Sure 👌